### ISSUE TITLE
SyntaxError: Invalid or unexpected token (again -#2902?)

### ISSUE THREAD

**Issue by celstark** [OpenNeuroOrg/openneuro#3069]
### What went wrong?

Trying to run `openneuro login` yields the following error:
```
/usr/lib/node_modules/@openneuro/cli/node_modules/bids-validator/dist/commonjs/index.js:20270
        useFipsEndpoint: options2.useFipsEndpoint ?? false,
                                                   ^

SyntaxError: Invalid or unexpected token
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)

Node.js v20.14.0
```

### Expected behavior

I expect it to prompt me for the login info

### How to reproduce

1. Install fresh npm and node.js
2. Install fresh @openneuro-cli
3. Type `openneuro login` or `openneuro --help`

### Desktop

- OS:
- Browser:
- Version:

### Phone

System: Zorin OS 17 (Ubuntu jammy based)
Code version: @openneuro/cli@4.24.0
Node.js version: 20.14.0 (also tried 22)

### Additional information

xref #2902

**Comment by effigies** [comment]
The last fix https://github.com/OpenNeuroOrg/openneuro/pull/2905 was to roll back the validator to 1.10.0.

**Comment by nellh** [comment]
Thanks for reporting this issue, it should be fixed in @openneuro/cli version 4.24.2. Please let us know if you're still having issues.

**Comment by celstark** [comment]
Working!
